forks/.editorconfig

```diff
@@ -0,0 +1,18 @@
+# Analyzers
+[*.cs]
```

Comment:
We've migrated to using `.globalconfig` for analyzer rules. Please use that instead with a higher `global_level` number like we do in healthcare-shared-components.

Eg.
- [root config](https://github.com/microsoft/healthcare-shared-components/blob/main/.globalconfig)
- [per-project config](https://github.com/microsoft/healthcare-shared-components/blob/main/src/Microsoft.Health.Operations.Functions/.globalconfig)

Reply:
Done.